portfolio/src/main/webapp/comments.js

```diff
@@ -93,6 +99,10 @@ function getCommentReplyForm(comment) {
 function createFormInput(type, name, placeholder, value = null, hidden = false) {
   const input = document.createElement('input');
   input.type = type;
+  // if type of the input is 'text', then it must have value before submitted
+  if (input.type === 'text') {
+    input.required = true;
```

Comment:
Does blank spaces get removed automatically?

Reply:
I mean to ask if I just provide white spaces will the required test pass?

Reply:
oh, indeed it will not